compiler/src/dmd/declaration.d

```diff
@@ -779,8 +779,17 @@ extern (C++) final class AliasDeclaration : Declaration
             /* Semantic analysis is already finished, and the aliased entity
              * is not overloadable.
              */
-            if (type)
-                return false;
+            if (type) {
```

Comment:
Allman style (shouldn't this be picked up by the linter?)

Reply:
@maxhaton ping

Reply:
done